8.gtex-interpret/scripts/nbconverted/2A.download-neutrophil-data.py

```diff
@@ -53,6 +53,7 @@
 
 # Load Data
 geo_df = pd.read_excel(path, index_col=0, skiprows=1)
+geo_df = geo_df[geo_df.symbol != 'N\A']
```

Comment:
is this the same as dropna?

Reply:
No, unfortunately this dataset had some strange symbols